### DISCUSSION TITLE
Is there an easier way to change accordion icon?

### DISCUSSION THREAD

**Discussion by DenisCor** [carbon-design-system/carbon#12936] [Help]
Accordion doesn't have a prop that will allow you to change the icon on the fly. Feels like if you want to change the default "accordion__arrow" svg you have to target that html element and do it in a weird/messy way..  Any alternatives?

**Comment by tay1orjones** [comment]
@DenisCor Accordion is built to adhere to the design specification, which [explicitly specifies chevrons](https://carbondesignsystem.com/components/accordion/style#structure). What are you wanting to change it to instead? Could you expand on why and the use case?
